addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -763,6 +763,8 @@ QUnit.test('chat window: state conservation on toggle home menu', async function
     // Show home menu
     await this.env.bus.trigger('will_show_home_menu');
     await this.env.bus.trigger('show_home_menu');
+    // in real condition, chat window will be unmount and recreated
+    await this.env.services['chat_window']._mount();
```

Comment:
My wishes are not what drived me to do that, but the triggering of these events is not enough to simulate correctly what happens in real situation. That's a problem because test was green but in real situation it didn't work at all, so the previous test was just totally useless IMO.

So I don't really know what's better but we should have something "as close as possible" of the real situation to make sure this kind of problem doesn't show up again

Reply:
Looks like integration test -> could be tour then

Reply:
Without me then.

Reply:
(This implies to manage to open messaging menu in tour andddd this is already a blocking issue shown by https://github.com/odoo/enterprise/pull/11535

Reply:
I would add prefix `[technical]` on name of test. Would be nice to replace explicit `_mount()` by changing target of `body`/`#qunit-fixture` content in a fragment before `show_home_menu` event, so that it matches home menu behaviour.

Reply:
How do you do that ? :open_mouth:

Reply:
Something like:
```js
$(`#qunit-fixture`).contents().appendTo(document.createDocumentFragment())
```